playground/stories.tsx

```diff
@@ -0,0 +1,16 @@
+import {Playground} from './Playground';
+import {KitchenSink} from './KitchenSink';
+import {DetailsPage} from './DetailsPage';
+
+// eslint-disable-next-line import/no-default-export, import/no-anonymous-default-export
```

Comment:
If we start using stories files more then we can ignore this in the eslint config for stories.tsx files.